File: src/main/java/com/jiangli/api/controller/LoginController.java
package com.jiangli.api.controller;

import com.jiangli.api.service.UserService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * @author Jiangli
 * @date 2018/3/15 16:09
 */
@RestController
@RequestMapping("/user")
public class LoginController extends BaseController {
    @Autowired
    private UserService userService;

    /**
     * http://localhost:8010/user/login
     * @return
     */
    @RequestMapping("/login")
    public String login() {
        log.debug("login...");
        return "ClientMain Hello "+userService.queryByUserId(1L);
    }
}

File: src/main/java/com/jiangli/api/controller/LoginController.kt
package com.jiangli.api.controller

import com.jiangli.api.service.UserService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

/**
 *
 *
 * @author Jiangli
 * @date 2018/5/11 11:36
 */
@RestController
@RequestMapping("/user")
 class LoginController : BaseController() {
    @Autowired
    var userService:UserService? = null

    /**
     * http://localhost:8010/user/login
     * @return
     */
    @RequestMapping("/login")
    fun login():String {
        log.debug("login33443...")
        return "ClientMain Hell5353反反复复付付付付 " + userService!!.queryByUserId(1L)
    }
}